### DISCUSSION TITLE
Welcome to visual-basic-getting-started Discussions!

### DISCUSSION THREAD

**Discussion by karenpayneoregon** [karenpayneoregon/visual-basic-getting-started#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by meires** [comment]
I found problem with Microsoft.NETCore.App, already restore NuGet but not resolve this problem

**Reply by karenpayneoregon** [reply]
You need to be very specific. If you read back your statement in the third person what would you think?

**Comment by DualBrain** [comment]
@karenpayneoregon Not necessarily on topic, but not totally off topic either... any interest in joining us VB folks over on a VB Discord server? https://discord.gg/Y8EH5fF6WG

**Reply by karenpayneoregon** [reply]
I joined but not sure of the group name, all I saw was Visual Basic Community, it that it?

**Reply by DualBrain** [reply]
I don't know that I've seen you on there (maybe you are on there by a name that I don't recognize). You should see a COMMUNITY category with general, help-each-other, show-your-stuff, etc. and several other categories that are specific to different editions of VB. The name of the server is simply "VisualBasic" and the glyph (icon?) is the blue circle with the letters VB in it.

**Reply by karenpayneoregon** [reply]
My user name is **karenpayneoregon** and in regards to VisualBasic server, tried to search for it but does not come up in a search.

**Reply by DualBrain** [reply]
I don't think it will show up in search until we have 1000 members. :-(

The direct invite link though is: https://discord.gg/Y8EH5fF6WG

**Reply by karenpayneoregon** [reply]
Thanks, I'm in.
